css-two-column-layout/styles.css

```diff
@@ -0,0 +1,42 @@
+.container {
+  max-width: 1050px;
+  margin-left: auto;
+  margin-right: auto;
+  font-family: sans-serif;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  align-content: center;
```

Comment:
I would move these out of the row class and create another class such as `justify-center` and `align-center` so you can apply those classes individually. A `row` class is typically just a horizontal section of the page.